### PR TITLE
Minor authorization page fixes styles

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
With this PR, I adjust the styles a little 3.x. This PR is focused on changing the visual representation of the authorization page.

I have corrected:
- centered and increased the size of the logo
- text indents, input field descriptions
- link colors, input field descriptions
- seems to have centered the position of the checkbox and the text =)
- font sizes
- button colors their hover behavior
- styles at the form input
- styles of error message and successful operation

### Why do you need it?
Redesign is one of the key features of MODX3 and I tried to bring the styles of this page a little closer to the overall design.

Before:
modx revolution 2019-01-26 05-59-49
modx revolution 2019-01-26 05-59-36

After:
login modx revolution - 2019-01-30 18-14-40
login modx revolution - 2019-01-30 18-15-14
login modx revolution - 2019-01-30 18-15-30

### Related issue(s)/PR(s)
#13948

### CODE REVIEW

File: _build/templates/default/sass/_colors-and-vars.scss [hunk 1]
```diff
@@ -19,8 +19,12 @@ $black: darken($colorSplash, 42.5%); /* generate the black from the $colorSplash
 $yellow: #FFEA80;
 $orange: #F49548;
 $red: #BE0000;
+$lighterRed: #FFEEEE;
 $green: #6CB24A;
+$lighterGreen: #EFFCF6;
+$greener: darken($green, 12%);
 $blue: #3697cd; /* default theme $colorSplash */
+$havelockBlue: #507dd7;
```
Comment (JoshuaLuckers) [inline _build/templates/default/sass/_colors-and-vars.scss]:
```suggestion
$havelockBlue: #507DD7;
```

Improve code quality by using capitalization consistently.

**Comment by christianseel** [comment]
I like the success/error styles! However the logo looks way too big for me now. I'm also not sure about the inputs as they look somehow disabled now?

Is the (button) green the same as the current primary color of the manager interface? Because at the time when we were working on the login, the previous color was used as primary in the manager.

**Comment by christianseel** [comment]
Ah, just found https://github.com/modxcms/revolution/pull/14306 which seems to be related and which streamlines the button colors 👍🏻

File: _build/templates/default/sass/_colors-and-vars.scss (hunk 1 above)
Comment (Jako) [reply _build/templates/default/sass/_colors-and-vars.scss]: Are there editor styles available for this?

**Comment by Ibochkarev** [comment]
> I'm also not sure about the inputs as they look somehow disabled now?

@christianseel I ask you to test these changes. Since there is always the possibility to make adjustments.

I will fix the logo. 

Thanks for the feedback. I tried to do in the same style.

File: _build/templates/default/sass/_colors-and-vars.scss (hunk 1 above)
Comment (Ibochkarev) [reply _build/templates/default/sass/_colors-and-vars.scss]: > Are there editor styles available for this?

@Jako in 137 line

Comment (Jako) [reply _build/templates/default/sass/_colors-and-vars.scss]: I meant by this file: https://github.com/modxcms/revolution/blob/3.x/.editorconfig

Comment (Jako) [reply _build/templates/default/sass/_colors-and-vars.scss]: In PhpStorm these styles could be predefined in https://www.jetbrains.com/help/phpstorm/settings-code-style-scss.html. Maybe there is a way to put those settings in the .editorconfig.

Comment (Ibochkarev) [reply _build/templates/default/sass/_colors-and-vars.scss]: Open #14311

**Review by alroniks — CHANGES_REQUESTED** [review]
After the quick look:
- New blue color seems to bold in comparing with old one. I'd prefer to return it.
- As some font properties were changed it's strongly required test it on Latin letters too (screenshots).
- I don't agree with replacing focus stroke by shadows. Availability is completely lost.

**Review by JoshuaLuckers — CHANGES_REQUESTED** [review]
Keep the key points in the original plan in mind when changing the login screen:
- [MODX 3 Plan - Login screen](https://modx3.org/plan/login-screen)
- [Another example of the login screen](https://modx3.org/updates/week-7)

**Comment by Mark-H** [comment]
@JoshuaLuckers We honestly don't have to stick to what was posted on modx3.org a year ago if a better proposal comes around. If you have specific points of feedback, please elaborate.

**Comment by JoshuaLuckers** [comment]
I agree that we don't have to stick to something that's proposed a long time ago. We do have to keep in mind the designs that where proposed. Deviations from this design should have proper argumentation (that's what I'm missing right now).

**Comment by Mark-H** [comment]
I agree with @Alroniks on the focus status (the inset box shadow is a little too subtle), and I'd probably take out the letter-spacing on the ghost buttons/password reset link. Makes it look a little stretched and inconsistent with other text on latin text:

schermafbeelding 2019-01-29 om 14 36 15

Those two things changed would earn my ✔️ . 

I don't really have an opinion on the blue color. I think it's an interesting shade of blue, but if it's not used anywhere else but the login page, it may make sense to restore that to something more consistent. 

@JoshuaLuckers: Design is very subjective of course, but in my opinion the screenshots made it clear that it adds more polish to the design, which is more than enough argumentation in my book ;)

**Review by alroniks — APPROVED** [review]
Now it looks much better! 👍

**Review by JoshuaLuckers — APPROVED** [review]

**Comment by Ibochkarev** [comment]
The last word is yours @Mark-H  =)

**Review by Mark-H — APPROVED** [review]

**Pull request merged**